### PR TITLE
Fixing GH workflow to check pods in accuknox-agents namespace as per …

### PR DESCRIPTION
**Pull request by seswarrajan**
…latest k8s deployment

Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

**Review by nyrahul — APPROVED** [review]

**Review by Ankurk99 — CHANGES_REQUESTED** [review]
Can you please change the label to reflect @yasin-cs-ko-ak changes?

**Comment by seswarrajan** [comment]
> Can you please change the label to reflect @yasin-cs-ko-ak changes?

Thanks for reminding me that

File: .github/workflows/latest-release.yml [hunk 1]
```diff
@@ -40,8 +40,7 @@ jobs:
 
       - name: Run discovery engine
         run: |
-            kubectl apply -f deployments/k8s/ && kubectl wait --for=condition=ready --timeout=5m -n default pod -l container=knoxautopolicy
-            kubectl get pods -A
+            kubectl apply -f deployments/k8s/ && kubectl wait --for=condition=ready --timeout=5m -n accuknox-agents pod -l app=discovery-engine            kubectl get pods -A
```
Comment (Ankurk99) [inline .github/workflows/latest-release.yml]: both commands got merged in the same line 😁

Comment (Ankurk99) [reply .github/workflows/latest-release.yml]: also pls squash the commits

**Review by Ankurk99 — APPROVED** [review]

**Pull request merged**